service/spi/src/main/java/gov/va/vro/service/spi/model/Claim.java

```diff
@@ -26,10 +27,19 @@ public class Claim {
 
   private UUID recordId;
 
-  @NotNull private String claimSubmissionId;
+  // Either benefitClaimId or CollectionId must be filled out
+  // CollectionId is equivalent to reference_id on the claim submission table
+  private String benefitClaimId;
 
   private String collectionId;
 
+  // Backwards compatibility with v1 routes. On the way in mappers put claimsubmission id into
+  // collection id
+  // Both ways tie to the reference_id on claim submission table.
+  public String getClaimSubmissionId() {
```

Comment:
Ideally there would be a v1 and a v2 claim entity object. However due to time constraints there is no possible way to do that.

Reply:
Oh, sorry. I apparently misworded that comment. I meant, a v1 and v2 Claim *spi model* that we pass around the code. You are absolutely correct we definitely do not want 2 claim entities or table.